warehouse/api/simple.py

```diff
@@ -21,6 +21,25 @@
 from warehouse.packaging.models import JournalEntry, Project
 from warehouse.packaging.utils import _simple_detail, _simple_index
 
+# Generate appropriate CORS headers for the JSON endpoint.
+# We want to allow Cross-Origin requests here so that users can interact
+# with these endpoints via XHR/Fetch APIs in the browser.
+_CORS_HEADERS = {
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Headers": ", ".join(
+        [
+            "Content-Type",
+            "If-Match",
+            "If-Modified-Since",
+            "If-None-Match",
+            "If-Unmodified-Since",
+        ]
+    ),
+    "Access-Control-Allow-Methods": "GET",
+    "Access-Control-Max-Age": "86400",  # 1 day.
+    "Access-Control-Expose-Headers": ", ".join(["X-PyPI-Last-Serial"]),
+}
+
```

Comment:
Since this is exactly the same as https://github.com/pypi/warehouse/blob/d42f64d56021ccf9e807dc72edf8eb589a9cbc17/warehouse/legacy/api/json.py#L26-L40, maybe we should find a way to de-duplicate instead?

Reply:
Maybe put this dictionary to `utils/cors.py` and import from there? I'm not sure where would be the best place to put this.